src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$block-size: 100px;
```

Comment:
Consider renaming the variable '$block-size' to something more descriptive, like '$calendar-cell-size', to improve code readability.